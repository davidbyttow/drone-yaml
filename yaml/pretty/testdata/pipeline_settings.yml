kind: pipeline
name: default

steps:
- name: notify
  image: plugins/slack
  settings:
    token:
      $secret: token
    root: general
    labels:
    - foo
    - bar
    - baz
