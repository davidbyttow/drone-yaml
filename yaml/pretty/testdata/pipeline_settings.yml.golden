---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: notify
  image: plugins/slack
  settings:
    labels:
    - foo
    - bar
    - baz
    root: general
    token:
      $secret: token

...
